### ISSUE TITLE
[bug] variable returning number | workflow | Sales Orders

### ISSUE THREAD

**Issue by waran70** [YetiForceCompany/YetiForceCRM#11257]
Wanna add Account name to subject in invoice, but workflow returning number instead of Account Name.... quick fix exists?
Tnx

**Comment by bpabiszczak** [comment]
Show me wideo.

**Comment by vovpff** [comment]
@bpabiszczak @waran70 I can confirm what in the older versions of Yeti workflow returns record CRMID instead Record Name

**Comment by z0lo13** [comment]
you can use custom function the method would be:

$relationid = $recordModel->get('leadid');
$relatedRecordModel = \Vtiger_Record_Model::getInstanceById($relationid);
$relationId = $relatedRecordModel->getName()?

**Comment by vovpff** [comment]
@z0lo13 where this code need to be put?

**Comment by z0lo13** [comment]
First of all the video is showing how you create an invoice and you want to have not an invoice but a quote it is different modules.
There is a result of my function:
![qtest](https://user-images.githubusercontent.com/23143958/63101497-240e2200-bf47-11e9-8503-0a567a90bd5c.gif)

I created a file for your qtest but for your production, you will need to modify that.

Place it in:
`/modules/SQuotes/workflows/`

[Qtest.zip](https://github.com/YetiForceCompany/YetiForceCRM/files/3505845/Qtest.zip)

you also need to create a new row in **com_vtiger_workflowtasks_entitymethod**
```
INSERT INTO `com_vtiger_workflowtasks_entitymethod` (`module_name`, `method_name`, `function_path`, `function_name`) VALUES ('Squotes', 'QtestProcess', 'modules/SQuotes/workflows/Qtest.php', 'Qtest');
```
For more info refer to:

#10667

**Comment by waran70** [comment]
z0lo13, there is always way around, but the point is it should work out of the box. Creating custom function just for such little thing is a bit too much :-)

**Comment by z0lo13** [comment]
> 
> 
> z0lo13, there is always way around, but the point is it should work out of the box. Creating custom function just for such little thing is a bit too much :-)

I know, but if you are ready to wait for weeks for such a small thing that fine.

Even if they fix this bug it will come in next update 5.2, otherwise in its next service packs, or you will need manually implement it in your code, which I don't think right things to do.

**Comment by adudekyeti** [comment]
Hi,
We do not see sense to do it in that way. This field is number field so given value is correct.
If you really want to have it, you can use JavaScript, to copy value to text field, when you select related record.
Or second way you can create new type of uitype on both modules. Then you can manage the values the way you choose.
